packages/commcare/src/Adaptor.js

```diff
@@ -82,6 +82,49 @@ export function get(path, params = {}, callback = s => s) {
   };
 }
 
+/**
+ * Make a post request to commcare
+ * @example
+ * post(
+ *   "idgen/identifiersource/8549f706-7e85-4c1d-9424-217d50a2988b/identifier",
```

Comment:
Is this a real url from commcare?

I'm not it's a great example, the empty object in particular isn't intuitive - is that data or options? Why is it empty either way?

User creation would seem like a simpler story:
```
post('user', (state) => state.data)
```